website/members/management/commands/get_mercadopago_payments.py

```diff
@@ -41,25 +53,34 @@ def process_mercadopago(self, results, filter_payment_id, filter_payer_id):
         payments = []
         for info in results:
 
-            if info['operation_type'] != 'recurring_payment':
+            # discard weird records about cards authorizations
+            if "payer" not in info:
+                logger.debug("Discarding non-payer record: %s", info)
                 continue
 
-            # needed information to record the payment
-            timestamp = parse_datetime(info['date_approved'])
-            amount = Decimal(info['transaction_amount'])
             payer_id = info['payer']['id']
             assert payer_id is not None
             payer_id = str(payer_id)
 
+            # only suscriptions are handled here
+            reason = info['description']
+            if reason not in SUBSCRIPTIONS:
```

Comment:
Dejo una pequeña sugerencia para no tener que venir a tocar el código cuando hagamos un nuevo aumento (siempre que respetemos el formato del nombre de las subscripciones 😅 )
```suggestion
            reason_is_subscription = (
                 # subscriptions until 2023
                 reason in SUBSCRIPTIONS
                 # Posible future subscriptions
                 or reason.startswith("Socies Adherentes cuota") 
                 or reason.startswith("Socies Actives cuota")
             )

                 
            if not reason_is_subscription:
```

Reply:
Me gusta. Pero quizás ir un paso más allá?

```
reason_is_subscription = reason.startswith(("Socies Adherentes cuota", "Socies Actives cuota", "Cuota mensual"))
```

Reply:
Me gusta! Aunque me parecía útil tener el listadito de las subscripciones actuales, para referencia. Quizás podríamos dejar algunos ejemplos en comments?

```python
# Some examples of subscriptions are: "Cuota mensual socio adherente 2018", "Socies Actives cuota 2023"
reason_is_subscription = reason.startswith(("Socies Adherentes cuota", "Socies Actives cuota", "Cuota mensual"))
```

La posta posta sería agregar test unitarios para asegurarnos que funciona con el listado de subscripciones conocidas, pero no hay ni un solo test así que el esfuerzo sería demasiado grande jeje 🙃

Reply:
Ahí pushé cambios